2_run/src/run_glm3.R

```diff
@@ -131,9 +157,13 @@ run_glm3_model <- function(sim_dir, nml_objs, model_config, export_fl_template)
         export_fl = export_fl,
         export_fl_hash = tools::md5sum(export_fl),
         glm_run_date = Sys.time(),
+        glm_version = GLM3r::glm_version(as_char = TRUE), #Needs version 3.1.18 of GLM3r
         glm_time_s = glm_time,
-        glm_success = TRUE,
-        glm_code = glm_code)
+        param_sim_start = sim_start,
+        param_sim_stop = sim_stop,
+        max_output_date = max_output_date,
+        glm_code = glm_code,
+        glm_success = TRUE)
```

Comment:
Is there anything from this list that should kick off a rebuild? And thus be added to [this list of columns](https://github.com/USGS-R/lake-temperature-process-models/blob/main/2_run.R#L21)? I don't think so - changes in `sim_start`, `sim_stop`, `glm_version` etc would make the datafiles themselves different which would be fine for kicking off a rebuild. Just thought it might be worth the mental exercise to double check.

Reply:
Good thought. I don't think we need to add anything, as our setting `export_fl` and `export_fl_hash` to NA if the run fails should catch all instances when the run failed, and if the run doesn't fail, any changes to the datafiles themselves should be caught by that `export_fl_hash` column, like you said.